frontend/packages/topology/src/components/side-bar/components/SideBarHeading.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { GraphElement } from '@patternfly/react-topology';
+import { useDetailsResourceLink } from '../providers/useDetailsResourceLink';
+import SideBarAlerts from './SideBarAlerts';
+import './SideBarHeading.scss';
+
+const SideBarHeading: React.FC<{ element: GraphElement }> = ({ element }) => {
+  const resourceLink = useDetailsResourceLink(element);
+  return (
+    <div className="overview__sidebar-pane-head resource-overview__heading">
+      <h1 className="co-m-pane__heading">
+        <div className="co-m-pane__name co-resource-item">{resourceLink}</div>
```

Comment:
There needs to be a fallback resource link if one is not supplied via extension.
1) extract the resource and create a ResourceLink
2) render a non-linkable text based on the element's label